### DISCUSSION TITLE
nix-flatpak moving under the nix-community umbrella.

### DISCUSSION THREAD

**Discussion by SpiderUnderUrBed** [nix-community/infra#1429] [General]
There has been a discussion about migrating nix-flatpak here, (until maybe, if, they migrate upstream to nixpkgs), the discussion is [here](https://github.com/gmodena/nix-flatpak/issues/69), This discussion is about if nix-flatpak has a place in nix-community and any further coordination, note I am not a contributer to nix-flatpak, I just opened this discussion as it was under consideration by the maintainers, and this discussion seeks to provide a place for the maintainers to talk if they still wish to consider porting efforts.

**Comment by gmodena** [comment]
Hi @SpiderUnderUrBed ,

I'm currently **not** looking for an ownership transfer of nix-flatpak. I have the time to maintain it and welcome bug reports, feature requests, PRs and other contributions at the project home.

If the situation changes, I'll definitely reach out.
